assets/quick-add.css

```diff
@@ -35,6 +35,18 @@
   transform: none;
 }
 
+.quick-add-modal .quick-order-list__container {
+  padding-bottom: 1.5rem;
+}
+
+.quick-add-modal .link.product__view-details {
+  text-decoration: none;
+}
+
+.quick-add-modal .link.product__view-details .link__text {
+  text-decoration: underline;
+}
```

Comment:
This seem a bit hacky to me 🤔 I think we can keep the original approach and add a margin-left to the svg arrow icon just like we do for the featured-product section: 
<details>
<summary>screenshot</summary>

![](https://screenshot.click/28-32-1ty0v-s96a4.png)
</details>